### ISSUE TITLE
iOS 9 displays separator lines incorrectly

### ISSUE THREAD

**Issue by danielsaidi** [danielsaidi/Sheeeeeeeeet#73]
On iOS 9, separator lines are incorrectly displayed leftmost with a width of ~15 points for items that are configured not to show separators (e.g. the example's titles, section titles and section margins). Also, a zero inset does not go all the way to the left.

**Comment by danielsaidi** [comment]
The insets are not honored due to a behavior in iOS 9.0, that requires you to configure the cells in ways you don't have to on newer versions on iOS. I have added the following to the `ActionSheetItemCell` class and calls it as the cell moves to window:

```swift
open func fixSeparatorLineBugOnOlderDevices() {
   if #available(iOS 10.0, *) {} else {
      preservesSuperviewLayoutMargins = false
      layoutMargins = .zero
   }
}
```

**Comment by danielsaidi** [comment]
This is fixed in `1.3.1`
